lib/Stats.js

```diff
@@ -323,6 +324,10 @@ Stats.prototype.toJson = function toJson(options, forToString) {
 			return obj;
 		});
 	}
+
+	if(showPerformance && compilation.compiler.options.target === "web") { 
```

Comment:
Whats the best way to access options and target via stats. I see the compilation is pulled in earlier.